src/control/controlpotmeter.cpp

```diff
@@ -146,76 +117,95 @@ PotmeterControls::PotmeterControls(const ConfigKey& key)
 PotmeterControls::~PotmeterControls() {
 }
 
+void PotmeterControls::addAlias(const ConfigKey& key) {
+    m_controlUp.addAlias(ConfigKey(key.group, QString(key.item) + QStringLiteral("_up")));
+    m_controlDown.addAlias(ConfigKey(key.group, QString(key.item) + QStringLiteral("_down")));
+    m_controlUpSmall.addAlias(ConfigKey(
+            key.group, QString(key.item) + QStringLiteral("_up_small")));
+    m_controlDownSmall.addAlias(ConfigKey(
+            key.group, QString(key.item) + QStringLiteral("_down_small")));
+    m_controlSetDefault.addAlias(ConfigKey(
+            key.group, QString(key.item) + QStringLiteral("_set_default")));
+    m_controlSetZero.addAlias(ConfigKey(
+            key.group, QString(key.item) + QStringLiteral("_set_zero")));
+    m_controlSetOne.addAlias(ConfigKey(key.group, QString(key.item) + QStringLiteral("_set_one")));
+    m_controlSetMinusOne.addAlias(ConfigKey(
+            key.group, QString(key.item) + QStringLiteral("_set_minus_one")));
+    m_controlToggle.addAlias(ConfigKey(key.group, QString(key.item) + QStringLiteral("_toggle")));
+    m_controlMinusToggle.addAlias(ConfigKey(
+            key.group, QString(key.item) + QStringLiteral("_minus_toggle")));
```

Comment:
this is duplicated across the constructor, I'm worried that we might forget to update both places in case we add another suffix, would this be possible to deduplicate?

Reply:
I don't really know how to do that in an elegant way. I pushed a little helper function to make the ConfigKey creation less verbose though.

Reply:
Well, I came up with this: https://compiler-explorer.com/z/xxE83Gf8r
I find it very elegant, just don't know how easy it is to understand.
Frankly, now that I've written it, I don't think we should use it here. Its very seldom that we would add another suffix for `ControlPotMeter`.

Reply:
Next best thing would just be a comment in both lists.

Reply:
https://github.com/mixxxdj/mixxx/pull/11998/commits/a3ca5eae6c50384650c5f6248f74e10efc6f9958